application/views/pc/pages/sunglasses/imagesByModelFlexSlider.php

```diff
@@ -57,6 +57,15 @@ class="width-75-percent"
                                 <?php endif; ?>
 								 </p>
 							</div> <!-- end .container -->
+							<div id="infograph">
+							  <div class="center-infograph">
+									<div id="img-1"></div>
```

Comment:
`img-1` is a bad id. ID must be unique on the entire html page. `img-1` doesn't look like a unique name.